src/status.rs

```diff
@@ -226,9 +237,23 @@ impl Status {
     }
 }
 
+#[derive(Debug, Clone)]
+enum StatusBuilderVersion {
+    V0_13,
+    V14,
+    Mixed,
+}
+
+impl Default for StatusBuilderVersion {
+    fn default() -> StatusBuilderVersion {
+        StatusBuilderVersion::V0_13
+    }
+}
```

Comment:
Does a default impl for the version make sense? I'd rather remove the `Default` derive from `StatusBuilder`.

Reply:
Well it would make constructing `StatusBuilder` a lot more annoying since one would need to specify all the fields (see `new()`, `v0_13()`, `v14()` currently line 273.
If we remove `Default` I'd also remove the `new` constructor.

Reply:
Yeah, true. But from a user perspective, as long as we use the constructor methods, it shouldn't be an issue, right?